### PR TITLE
Default training setup for new cluster

### PR DESCRIPTION
**Pull request by tilmankamp**
- Based on 0.1.1 hyperparameters
- Successfully started a test training 
- This will/should not get merged before test training concludes

X-DeepSpeech: NOBUILD

### CODE REVIEW

File: .compute [hunk 1]
```diff
@@ -1,89 +1,29 @@
...
+  --learning_rate 0.0001 \
+  --dropout_rate 0.2367 \
+  --default_stddev 0.046875 \
+  --epoch 13 \
+  --display_step 0 \
+  --validation_step 1 \
+  --checkpoint_step 1 \
+  --log_level 0 \
+  --checkpoint_dir "../keep" \
+  --summary_dir "../keep/summaries" \
```
[the remark this answers is not in the document]
Comment (tilmankamp) [reply .compute]: Yes - see `.install`

File: .compute [hunk 2]
```diff
@@ -1,89 +1,27 @@
...
+swb="${data}/LDC/LDC97S62/swb"
+lbs="${data}/OpenSLR/LibriSpeech/librivox"
+
+python3 -u DeepSpeech.py \
+  --train_files "${fis}-train.csv","${swb}-train.csv","${lbs}-train-clean-100.csv","${lbs}-train-clean-360.csv","${lbs}-train-other-500.csv" \
+  --dev_files "${lbs}-dev-clean.csv"\
+  --test_files "${lbs}-test-clean.csv" \
+  --train_batch_size 12 \
+  --dev_batch_size 8 \
+  --test_batch_size 8 \
```
Comment (reuben) [inline .compute]: I've been using {train,dev,test} = {24,48,48} on the cluster with the new code.

File: .compute [hunk 3]
```diff
@@ -1,89 +1,27 @@
...
+  --learning_rate 0.0001 \
+  --dropout_rate 0.2367 \
+  --epoch 13 \
+  --display_step 0 \
+  --validation_step 1 \
+  --log_level 0 \
+  --checkpoint_dir "../keep" \
+  --summary_dir "../keep/summaries" \
+  --decoder_library_path "../tmp/native_client/libctc_decoder_with_kenlm.so" \
+  "$@"
```
Comment (reuben) [inline .compute]: Also, this is probably useless with the new setup?

Comment (tilmankamp) [reply .compute]: What exactly?

Comment (reuben) [reply .compute]: The final parameter, "$@".

File: .compute (hunk 2 above)
Comment (tilmankamp) [reply .compute]: What do you mean by "the new code"? The code already in master?

File: .compute (hunk 3 above)
Comment (tilmankamp) [reply .compute]: Yeah - pretty much. Will remove it.

**Comment by tilmankamp** [comment]
First epochs of another test with updated batch sizes done - so they are no problem.

**Pull request merged**